index.html

```diff
@@ -20,14 +20,20 @@
 <body>
     <header class="nav-container">
         <div class="navbar-list">
-            <a href="#">
-                <h3 class="name1">AB</h3>
-            </a>
+            <h3 class="name1"><a href="#">AB</a></h3>
             <i class="hamburger-menu fa fa-bars"></i>
+            <ul class="hamburger-list">
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+                <li>
+                    <a href="#"><i class="fa fa-envelope-o"></i> </a>
+                </li>
+            </ul>
         </div>
     </header>
     <section class="header-container-top">
-        <div class="ellipse"></div>
+        <div class="lgImg1"><img src="./images/lg-img1.svg" alt="#" /></div>
         <div class="header-container">
             <div class="content">
                 <div class="personal-description">
```

Comment:
- [ ] This section should be centered in the desktop view it's aligned more to the left, As the previous reviewer suggests kindly try to center the text on the headline section and it's preferable to add some more font size to the `h1` so your design looks like the Figma one

![Screenshot_20220708_043913](https://user-images.githubusercontent.com/25227022/177912295-1254f266-2189-4fcb-a247-fad8da73f533.png)

Reply:
Hey @Ismailco

I have also fixed this error it works perfectly in my advice, would you mind checking once again and tell me which screen size works perfectly in all screen sizes on my side thanks.